ibtk/src/lagrangian/fortran/lagrangian_interaction2d.f.m4

```diff
@@ -750,31 +783,22 @@ c     Compute the interpolation weights.
 c
          do ic0 = ic_lower(0),ic_upper(0)
             X_cell(0) = x_lower(0)+(dble(ic0-ilower0)+0.5d0)*dx(0)
-            w0(ic0-ic_lower(0)) =
+            w(0,ic0-ic_lower(0)) =
      &           lagrangian_piecewise_cubic_delta(
```

Comment:
Here is an example of a place where we may want to replace the function call with a macro to facilitate vectorization, or to make the function process a vector of locations all at once.

Reply:
We should definitely do something (either put this function in the same translation unit or find out if using fortran modules enables inlining) to make it possible to inline but I am not convinced that we should vectorize this way.

Reply:
Does it make sense to replace the conditionals with floating point inequalities in the kernel function definition (for instance, using SIGN)?

Reply:
We'll have to avoid branches, so we will have to do something like that.